test/sanity/sanity_test.go

```diff
@@ -66,7 +66,7 @@ func TestSanity(t *testing.T) {
 
 	//Initialize GCE Driver
 	identityServer := driver.NewIdentityServer(gceDriver)
-	controllerServer := driver.NewControllerServer(gceDriver, cloudProvider, 1*time.Millisecond, 5*time.Minute)
+	controllerServer := driver.NewControllerServer(gceDriver, cloudProvider, 0*time.Millisecond, 5*time.Minute)
```

Comment:
just 0 is sufficient I think

Reply:
Done, and here is the passing sanity test after that change: https://prow.k8s.io/view/gs/kubernetes-jenkins/pr-logs/pull/sigs.k8s.io_gcp-compute-persistent-disk-csi-driver/1094/pull-gcp-compute-persistent-disk-csi-driver-sanity/1598452725211729920/